code/modules/antagonists/heretic/knowledge/blade_lore.dm

```diff
@@ -83,24 +94,21 @@
 		are_we_behind = TRUE
 
 	// We'll also assume lying down is behind, as mob directions when lying are unclear
-	if(target.body_position == LYING_DOWN)
-		are_we_behind = TRUE
+	if(isliving(target))
+		var/mob/living/living_target = target
+		if(living_target.body_position == LYING_DOWN)
+			are_we_behind = TRUE
 
 	// Exceptions aside, let's actually check if they're, yknow, behind
 	var/dir_target_to_source = get_dir(target, source)
 	if(target.dir & REVERSE_DIR(dir_target_to_source))
 		are_we_behind = TRUE
 
 	if(!are_we_behind)
```

Comment:
You don't need this `are_we_behind` var since you separated it out into its own proc, you should just replace it with `return TRUE` / `return FALSE`